services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js

```diff
@@ -220,10 +278,10 @@ export class ProcessPermit extends Component {
       <Menu.Item key="issue-permit" onClick={this.openIssuePermitModal} disabled={validationErrors}>
         Issue permit
       </Menu.Item>
-      <Menu.Item key="reject-application" onClick={this.openRejectApplicationModal}>
+      <Menu.Item key="reject-application" onClick={() => this.openRejectApplicationModal("REJ")}>
         Reject application
       </Menu.Item>
-      <Menu.Item key="withdraw-application" onClick={this.openWithdrawApplicationModal}>
+      <Menu.Item key="withdraw-application" onClick={() => this.openRejectApplicationModal("WDN")}>
```

Comment:
The logic in the called function will still work, but it's using "WDL" as a withdrawn status, and not "WDN".

Reply:
updated